### ISSUE TITLE
Focus indicator not visible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by seblington** [statelyai/editor-feedback#259]
Dear team,

I was just trying out creating a new state machine / process flow diagram, and I noticed that in High Contrast Mode, I can't see which button is currently focused when using keyboard only.

Without high contrast mode, the focus ring on buttons works fine.

Thanks for all the great work on Finite State Machines!
---
Studio version: 0.134.0

**Comment by davidkpiano** [comment]
Hey, this will be available in the next version of the editor. Stay tuned; coming soon!
